crates/server-primitives/src/admin.rs

```diff
@@ -96,7 +96,7 @@ pub struct NodeChallenge {
 #[serde(rename_all = "camelCase")]
 pub struct NodeChallengeMessage {
     pub nonce: String,
-    pub application_id: String, //optional if challenge is used on admin level
+    pub context_id: Option<calimero_primitives::context::ContextId>,
```

Comment:
is this safe to do now?

Reply:
Why wouldn't it be safe? Will update [this](https://github.com/calimero-network/core/blob/master/packages/calimero-sdk/src/wallets/NearLogin/NearLogin.tsx#L303) and other parts of sdk to replace application id

Reply:
Not sure why it was only a comment before

Reply:
As we don't need application_id if we login from admin dashboard as it doesn't have appid or contextid. Currently we just pass "admin-ui" for this value.